opentracing/span.lua

```diff
@@ -28,6 +29,17 @@ function span_methods:context()
   return opentracing_span_context.new()
 end
 
+-- Provides access to the @class `Tracer` that created this span.
+--
+-- @return the @class `Tracer` that created this span.
+function span_methods:tracer()
+  return self.tracer_
+end
+
+-- Changes the operation name.
+function span_methods:set_operation_name(operation_name)
+end
```

Comment:
Implementation?

Reply:
It's a noop implementation that documents the interface.

Reply:
Oh right OK makes sense.